playbooks/provision.yml

```diff
@@ -48,12 +48,16 @@
       db_user_roles: CREATEDB
 
     - role: geerlingguy.certbot
-      become: yes
+      certbot_install_from_source: yes
       certbot_auto_renew_user: "{{ unicorn_user }}"
-      when: protocol == "https"
-      tags: cert
-
-    - role: cert
+      certbot_create_if_missing: yes
+      certbot_create_method: standalone
+      certbot_create_standalone_stop_services: []
```

Comment:
we should add `nginx` here, right?

Reply:
I think the *standalone* runs only once when cert doesn't exist yet. In playbooks here nginx gets installed after later one. I think only the renewal command might need to use --nginx since it shouldn't use *standalone* any more but rely on certbot *nginx* plugin. In that case nginx using ports 80 and 443 makes sense since certbot should not try to spin a standalone we server any more.